configs/config.json

```diff
@@ -1,15 +1,16 @@
 {
     "publicAddress": "0x0000000000000000000000000000000000000000",
     "gasMax": 10,
-    "profitThreshold": 100,
     "dbFile": "tmp/db",
-    "configFolder": "configs",
+    "apiFile": "configs/api.json",
+    "manualDataFile": "configs/manualData.json",
     "envFile": ".env",
     "Mine": {
         "ListenHost": "0.0.0.0",
         "ListenPort": 9090,
         "RemoteDBHost": "localhost",
-        "RemoteDBPort": 5000
+        "RemoteDBPort": 5000,
+		"profitThreshold": 100
```

Comment:
we don't want this here as this will enable profit checking which we don't want be default